docs/release_notes/0.4.0.md

```diff
@@ -1,10 +1,24 @@
 # Release 0.4.0
 
 ## Features
-- ...
+
+Includes all features of the previous version, plus:
+- New experimental feature `generate profile` (#1132)
+- Added installation of Helm/Tiller as part of the `install flux` operation (#1136)
 
 ## Bug Fixes and code improvements
 - perform defaulting and validation in all commands (#1124)
 - improve nodegroup validation to detect duplicate names (#1116)
 - tidy up command usage formatting (#1112)
 - various internal improvements (#1113, #1117, #1118, #1119)
+- remove the ability of skipping tasks (#1101)
+- added tests for experimental features
+- refactored `install flux` experimental feature for testability
+- added common AWS CLI options to `install flux`
```

Comment:
This is also unimportant, but if you insist on adding it do mention the PR and please don't use "AWS CLI" (which was used in PR tittle, but it's actually misleading, as "AWS CLI" stands for [something else](https://aws.amazon.com/cli/).

```suggestion
- add common AWS flags to `eksctl install flux` (#1082) 
```

Reply:
This comment should be on commit descriptions too, then. They mention AWS CLI.